policy_packs/aws/stack/deploy_aws_iam_stack/stack/source.tofu

```diff
@@ -0,0 +1,26 @@
+variable "trusted_principals" {
+  description = "A list of principal ARNs that can assume the role. These wil be added to the trust policy."
+  type        = set(string)
+}
+
+resource "aws_iam_role" "read_only_role" {
+  name = "read_only_role"
+
+  assume_role_policy = jsonencode({
```

Comment:
I think we should use the `EOT` format instead of jsonencode. E.g. https://github.com/turbot/guardrails-samples/blob/main/policy_packs/aws/cis_v300/section1/policies.tf#L272.